flytekit/extras/tasks/shell.py

```diff
@@ -109,6 +110,8 @@ def __init__(
         task_config: T = None,
         inputs: typing.Optional[typing.Dict[str, typing.Type]] = None,
         output_locs: typing.Optional[typing.List[OutputLocation]] = None,
+        env: typing.Optional[typing.Dict[str, typing.Any]] = None,
```

Comment:
If python were more expressive, it wouldn't actually be Any here right? Its like "classes whose __str__ are valid when interpolated into bash"? This is probably fine as is, as I think there just isn't a way to effectively communicate that (at least that I can think of?)